### PR TITLE
fix: Cascader Attributes options Loss of responsiveness

### LINKED ISSUE #3455: [Bug Report] Cascader Attributes options Loss of responsiveness
### Element Plus version
1.1.0-beta.12

### OS/Browsers version
Chrome Version 93.0.4577.82 (Official Build) (64-bit)

### Vue version
3.2.11

### Reproduction Link
https://codepen.io/Aliang2021/pen/mdwpYvo?editors=1010

### Steps to reproduce
- before mounting cascader, initialize the options(Cascader Attributes)  properoty,   
like this: [ { value: 1, label: '1', children: [] } ]

- aftermounting, update the options properoty, sush as add something to the “children” array

- check if the view has new options
  
you can view my Reproduction Link above

--- plus ---  
After I downgrage the Element Plus version to 1.1.0-beta.10, I can't reproduce according to the same steps

### What is Expected?
I can see the newly added option on the view

### What is actually happening?
The data “options” has changed, but the view is not updated

### LINKED ISSUE #3597: [Bug Report] el-cascader: starting from v.1.1.0-beta.11 the options are not reactive anymore.
### Element Plus version
1.1.0-beta.15

### OS/Browsers version
Version 93.0.961.52 (Official build) (64-bit)

### Vue version
3.2.16

### Reproduction Link
https://jsfiddle.net/aq89eztw/2

### Steps to reproduce
Click on the  1 button. 

### What is Expected?
The list of the el-cascader options should grow.

### What is actually happening?
It does not grow.

image

### PR DESCRIPTION
**Pull request by Alanscut**
revert the changes of cascader in pr#3350 and edit guide word of props.multiple
fix: #3455 

Please make sure these boxes are checked before submitting your PR, thank you!

- [x] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [x] Make sure you are merging your commits to `dev` branch.
- [x] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

File: packages/components/cascader-panel/src/index.vue [hunk 1]
```diff
@@ -128,8 +128,8 @@ export default defineComponent({
         }
       }
 
-      oldConfig = configTemp
-      oldOptions = options
+      oldConfig = JSON.parse(JSON.stringify(configTemp))
```
Comment (sxzz) [inline packages/components/cascader-panel/src/index.vue:131]: Maybe we can use `_.cloneDeep` from lodash.

Comment (ZhenXiongQian) [reply packages/components/cascader-panel/src/index.vue]: ok

Comment (Alanscut) [reply packages/components/cascader-panel/src/index.vue]: it changed

File: packages/components/cascader-panel/src/index.vue [hunk 2]
```diff
@@ -26,6 +26,7 @@ import {
   watch,
 } from 'vue'
 import isEqual from 'lodash/isEqual'
+import _ from 'lodash'
```
Comment (sxzz) [inline packages/components/cascader-panel/src/index.vue:29]:
```suggestion
import cloneDeep from 'lodash/cloneDeep'
```

Comment (Alanscut) [reply packages/components/cascader-panel/src/index.vue]: Thanks!

**Review by sxzz — APPROVED** [review]

File: packages/components/cascader-panel/src/index.vue [hunk 3]
```diff
@@ -127,8 +128,8 @@ export default defineComponent({
         }
       }
 
-      oldConfig = configTemp
-      oldOptions = options
+      oldConfig = cloneDeep(configTemp)
+      oldOptions = cloneDeep(options)
```
Comment (SimonaliaChen) [inline packages/components/cascader-panel/src/index.vue:132]: This fix make no sense and will reproduce #3350

**Review by SimonaliaChen — CHANGES_REQUESTED** [review]

File: packages/components/cascader-panel/src/index.vue (hunk 3 above)
Comment (Alanscut) [reply packages/components/cascader-panel/src/index.vue:132]: what did you mean "will reproduce #3350"?
image
image

Comment (Alanscut) [reply packages/components/cascader-panel/src/index.vue:132]: ok,i will check it

File: docs/en-US/component/cascader.md [hunk 4]
```diff
@@ -44,7 +44,7 @@ cascader/last-level
 
 ## Multiple Selection
 
-Set `props.multiple = true` to use multiple selection.
+Add`:props="props"`in tag and set `props : { multiple = true }` to use multiple selection.
```
Comment (JeremyWuuuuu) [inline docs/en-US/component/cascader.md:48]: I think here might be a little confused for me personally. We should use a code block here to pointing out that what should be done, what should not. E.g.

Do:
```vue
<template>
  <el-cascader :props="props"/>
</template>

<script lang="ts">
export default {
  setup() {
      return {
          props: { // props.
             multiple: true,
          },
      }
  }
}
</script>

```

Don't do

```vue
<template>
  <!--  Object literal binging here is invalid syntax for cascader  -->
  <el-cascader :props="{ multiple: true}"/>
</template>
```

Comment (Alanscut) [reply docs/en-US/component/cascader.md:48]: @JeremyWuuuuu 
ok
btw，i found there are different languages blocks in [](https://element-plus.org/) but ,i just find en-US in the dev brance of element-plus. how can i change the other language guide

File: docs/en-US/component/cascader.md [hunk 5]
```diff
@@ -44,7 +44,36 @@ cascader/last-level
 
 ## Multiple Selection
 
-Add`:props="props"`in tag and set `props : { multiple = true }` to use multiple selection.
+Add`:props="props"`in tag and set data `props : { multiple = true }` to use multiple selection.<br/>
+Example:<br/>
+You have to use it like this:
```
Comment (JeremyWuuuuu) [inline docs/en-US/component/cascader.md:49]: Actually let's remove the inline `<br/>` tag, you can insert an empty line after each line to the the desired result.

The title can be simple as I mentioned, `Do` `Don't do` 

line#48 should be removed.

File: docs/en-US/component/cascader.md [hunk 6]
```diff
@@ -44,7 +44,36 @@ cascader/last-level
 
 ## Multiple Selection
 
-Add`:props="props"`in tag and set `props : { multiple = true }` to use multiple selection.
+Add`:props="props"`in tag and set data `props : { multiple = true }` to use multiple selection.<br/>
```
Comment (JeremyWuuuuu) [inline docs/en-US/component/cascader.md:47]:
```suggestion
Add`:props="props"`in tag and set data `props: { multiple = true }` to use multiple selection.

```

File: docs/en-US/component/cascader.md [hunk 7]
```diff
@@ -44,7 +44,36 @@ cascader/last-level
...
+  }
+</script>
+```
+
+Please don‘t use it like this:
+
+```html
+<template>
+  <!--  Object literal binging here is invalid syntax for cascader  -->
+  <el-cascader :props="{ multiple: true}" />
```
Comment (JeremyWuuuuu) [inline docs/en-US/component/cascader.md:75]:
```suggestion
<el-cascader :props="{ multiple: true }" />
```

File: docs/en-US/component/cascader.md [hunk 8]
```diff
@@ -44,7 +44,36 @@ cascader/last-level
 
 ## Multiple Selection
 
-Set `props.multiple = true` to use multiple selection.
+Add `:props="props"` in tag and set data `props: { multiple = true }` to use multiple selection.
```
Comment (JeremyWuuuuu) [inline docs/en-US/component/cascader.md:47]: One last request (I am sorry that I didn't bring it up earlier)
```suggestion
Add `:props="props"` in tag and set data `props = { multiple: true }` to use multiple selection.
```

Comment (JeremyWuuuuu) [reply docs/en-US/component/cascader.md:47]: You can simply apply my suggestion so that you don't have to update the code locally

**Comment by Alanscut** [comment]
@JeremyWuuuuu @SimonaliaChen 
The code changes in this pr are covered by a new pr on October 31st.
Now this pr only contains instructions for users, please decide whether to merge

**Pull request closed without merging**